backend/check.sh

```diff
@@ -3,13 +3,13 @@
 . include/common.sh
 . .envrc
 
-curl -s -Lo kube-ready-state-check.sh "$SCF_REPO"/raw/"$SCF_BRANCH"/bin/dev/kube-ready-state-check.sh
-chmod +x kube-ready-state-check.sh
-mv kube-ready-state-check.sh bin/
+curl -sSLo bin/kube-ready-state-check.sh \
+     https://raw.githubusercontent.com/cloudfoundry-incubator/kubecf/master/dev/kube/kube-ready-state-check.sh
```

Comment:
Hardcoding this instead of using SCF_REPO is painful.

But creating a new KUBECF_REPO is out of the scope for this PR, we really need to create `modules/kubecf/` at some point.

If we were to have KUBECF_REPO and SCF_REPO, we would need to have an `if kubecf then` at this point in time, when importing clusters, which also smells, as clusters shouldn't care about kubecf or scf.

We could run both kube-ready-state-check.sh, and only fail if both are failing (hence missing Tiller on kubecf will not be an issue).